src/main/webapp/i18n/de/textAssessment.json

```diff
@@ -47,12 +47,13 @@
             "cancel": "Bewertung abbrechen",
             "confirmCancel": "Bist du sicher, dass du die Bewertung abbrechen möchtest? Dein aktueller Bewertungsfortschritt wird gelöscht und die Abgabe steht anderen Tutor:innen wieder zur Verfügung.",
             "feedbackEditor": {
-                "dismissFeedback": "Möchtest du das Feedback löschen?",
-                "dismissFeedbackConfirmation": "Bitte bestätige, dass du das Feedback löschen möchtest!",
-                "feedbackCommentPlaceholder": "Gib deine Anmerkung bitte hier ein.."
+                "dismissFeedback": "Soll das Feedback gelöscht werden?",
+                "dismissFeedbackConfirmation": "Bitte bestätigen Sie, dass das Feedback gelöscht werden soll!",
+                "feedbackCommentPlaceholder": "Geben Sie Ihre Anmerkung bitte hier ein ...",
+                "impactWarning": "Dieses Feedback kann <strong style=\"color:blue\">{{affectedSubmissionsCount}}</strong> weitere ähnliche Einreichung/en beinflussen.",
+                "impactWarningTooltip": "Die Anzahl, wie oft Ihr Feedback als automatischer Vorschlag in anderen Einreichungen wiederverwendet wird, die einen ähnlichen Textblock aufweisen."
```

Comment:
We use `du` over `Sie` everywhere in the application, so imho this should stay/use `du`, `deine` etc. as well.